test/integration/buildruns_to_taskruns_test.go

```diff
@@ -75,14 +75,14 @@ var _ = Describe("Integration tests BuildRuns and TaskRuns", func() {
 			_, err = tb.GetBRTillStartTime(buildRunObject.Name)
 			Expect(err).To(BeNil())
 
-			err = tb.GetTRTillDesiredReason(buildRunObject.Name, "Pending")
-			Expect(err).To(BeNil())
+			actualReason, err := tb.GetTRTillDesiredReason(buildRunObject.Name, "Pending")
+			Expect(err).To(BeNil(), fmt.Sprintf("failed to get desired reason, we got %s", actualReason))
```

Comment:
nit - include the reason we were expecting:
```suggestion
			Expect(err).To(BeNil(), fmt.Sprintf("failed to get desired reason; expected %s, got %s", "Pending", actualReason))
```